eng/build.sh

```diff
@@ -262,6 +262,8 @@ while [[ $# > 0 ]]; do
           os="FreeBSD" ;;
         osx)
           os="OSX" ;;
+        maccatalyst)
+          os="MacCatalyst" ;;
```

Comment:
should `TargetArchitecture` be set to `arm64` for `MacCatalyst`?

Reply:
Mac Catalyst runs on macOS platforms (x64 or ARM64)